ocaml/xapi/sparse_dd_wrapper.ml

```diff
@@ -92,10 +92,8 @@ let dd_internal progress_cb base prezeroed verify_cert ?(proto = None) infile
                 match proto with
                 | None ->
                     []
-                | Some (StreamCommon.Nbd export) ->
+                | Some (`NBD export) ->
```

Comment:
If you want to change this, also change what is in `streamCommon.ml` the `type protocol` and also `dummy_nbd`

Reply:
That is in `vhd-tool`, not XAPI though, and there are other protocols there as well (Chunked/Human/Tar/etc.). I'd leave that alone for now.
It is fine for vhd-tool to support more protocols, but XAPI doesn't use them.